### ISSUE TITLE
IdsDataGrid: VirtualScroll Follow up issues

### ISSUE THREAD

**Issue by tmcconechy** [infor-design/enterprise-wc#1088]
**Describe the bug**
Listing some tasks we can do after #860 is merged.

**Tasks**

**Major**
- [x] Make sure scroll event fires for external devs using it... see https://github.com/infor-design/enterprise-wc/issues/1102
- [x] https://github.com/infor-design/enterprise-wc/issues/1100

**Medium**
- [ ] Fix new issue where scrolling down to 70 it "jumps"
- [ ] Move the logic to virtual scroll component so can be reused there

**Minor**
- [x] Scroll slowly down from top to bottom at a certain point the scroll bar "jumps" as the padding-bottom goes from 2000px to about 0px. This looks pretty janky in Firefox
- [ ] Add test coverage (Minor)
- [ ] Add a suppressCaching setting (setting added but needs to be wired in). It should immediately render the rows and cells with no cache. Default is true (Minor)
